### DISCUSSION TITLE
Connecting fabmanager with trotec ruby

### DISCUSSION THREAD

**Discussion by almostserena** [sleede/fab-manager#461] [General]
Hello,

Is there a lab or someone who connected the fabmanager with the hardware machines to allow users to unlock the machine only if they pay and if the slots are not paid the machine stops or go in pause? (or any similar way?)

**Comment by gnepud** [comment]
Hello @almostserena,

I apologize, but currently, we do not have a way to connect the fabmanager with the hardware machine. Perhaps we will develop this feature in the future.

Thanks
